src/GitHub.Exports/Helpers/SettingsStore.cs

```diff
@@ -0,0 +1,73 @@
+using System.IO;
+using Microsoft.VisualStudio.Settings;
+
+namespace GitHub.Helpers
+{
+    internal class SettingsStore
+    {
+        readonly WritableSettingsStore store;
+        readonly string root;
+        public SettingsStore(WritableSettingsStore store, string root)
+        {
+            this.store = store;
```

Comment:
I noticed we're not using `NullGuard` for this project so we should probably validate that these arguments are not null here. Otherwise, we could get a `NullReferenceException` on line 34.